pygdf/dataframe.py

```diff
@@ -426,8 +426,8 @@ def drop_column(self, name):
 
     @classmethod
     def _concat(cls, objs, ignore_index=False):
-        if len(set(o.columns for o in objs)) != 1:
-            what = set(o.columns for o in objs)
+        if len(set([tuple(set(o.columns)) for o in objs])) != 1:
```

Comment:
Would love something cleaner than this, but haven't come up with anything thus far.